packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -54,6 +58,21 @@ export function generateIsomorphicTests(
                         // errors will fail the test and log full stack traces to the console. nifty!
                         Enzyme.render(element);
                     });
+                    it(`<${componentName}> className is applied to outer element`, () => {
+                        const element = React.createElement(
+                            Component,
+                            {
+                                ...props[componentName],
+                                className: "test",
+                            },
+                            children[componentName],
+                        );
+                        if (classNameChildList.includes(componentName)) {
+                            assert.isAtLeast(Enzyme.render(element).find(".test").length, 1);
```

Comment:
looks like this change caused some tests to fail, but this is definitely more correct.

could add `{ inline: true }` prop to those components to keep everything inside the container element.

Reply:
It seems like this gets a little tricky for a few components. Alert and Dialog render className on a child div within an Overlay element, but that div isn't rendered unless `isOpen: true` is passed as a prop. But if `isOpen: true` is passed in, then we get an error about "document not defined." Is there an easy way to mock the document for these components to attach to?

Reply:
aw nuts. there is certainly not an easy way to mock the document. i can take a look at this.